tests/system/Helpers/InflectorHelperTest.php

```diff
@@ -95,15 +95,51 @@ public function testPlural()
 	public function testCounted()
 	{
 		$triplets = [
-			[3, 'cat', '3 cats'],
-			[1, 'cat', '1 cat'],
-			[0, 'cat', '0 cats'],
-			[3, 'cats', '3 cats'],
-			[1, 'cats', '1 cat'],
-			[0, 'cats', '0 cats'],
-			[3, 'fish', '3 fish'],
-			[1, 'fish', '1 fish'],
-			[0, 'fish', '0 fish'],
+			[
```

Comment:
If you can revert this one change, I'll merge.

Reply:
reverted.